TestProjects/Playground/Playground.iOS/Setup.cs

```diff
@@ -14,8 +14,8 @@ public Setup(MvxApplicationDelegate applicationDelegate, UIWindow window)
         {
         }
 
-        public Setup(MvxApplicationDelegate applicationDelegate, IMvxIosViewPresenter presenter)
-            : base(applicationDelegate, presenter)
+        public Setup(MvxApplicationDelegate applicationDelegate, UIWindow window, IMvxIosViewPresenter presenter)
```

Comment:
The problem with this was that in this case Window was never set, so it was never working. In any case, i've never used this ctor, but always the other one.